src/PhpSpreadsheet/Style/ConditionalFormatting/ConditionalFormattingRuleExtension.php

```diff
@@ -80,9 +80,12 @@ public static function parseExtLstXml($extLstXml)
 
                     $extDataBarObj = new ConditionalDataBarExtension();
                     $extFormattingRuleObj->setDataBarExt($extDataBarObj);
-                    $dataBarXml = $extCfRuleXml->dataBar;
-                    self::parseExtDataBarAttributesFromXml($extDataBarObj, $dataBarXml);
-                    self::parseExtDataBarElementChildrenFromXml($extDataBarObj, $dataBarXml, $ns);
+                   
+                    if ($dataBarXml = $extCfRuleXml->dataBar)
```

Comment:
It's not really considered good practise to apply an assignment within an `if` test, especially when it's using a loose coercion for truthiness.